CONTRIBUTING.md

```diff
@@ -0,0 +1,90 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+Discord, or any other method with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Ensure any installation or build dependencies are removed before the end of the layer when doing a
+   build.
+2. Update the README.md (of specific package/app) with details of changes to the interface; this includes new environment
+   variables, exposed ports, useful file locations and container parameters.
+3. You may merge the Pull Request in once you have the sign-off of Core Team developers, or if you
+   do not have permission to do that, you may request the Core Team member to merge it for you.
+
+## Code of Conduct
+
+### Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to make participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
+orientation.
+
+### Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+  advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
```

Comment:
Positive, constructive behavior is essential in fostering an enriching environment. Examples of such conduct include:

- Utilizing welcoming and inclusive language.
- Demonstrating respect for differing viewpoints and experiences.
- Gracefully accepting and offering constructive criticism.
- Prioritizing the well-being and development of the community.
- Exercising empathy towards other community members and respecting their personal boundaries.

Certain behaviors are considered unacceptable in our community. Inappropriate behaviors in our community include, but are not limited to: disrespectful language and actions, unwelcome attention or advances, harassment in any form, unauthorized disclosure of private information, and any other conduct deemed inappropriate in a professional setting.